sdcm/cluster.py

```diff
@@ -302,41 +301,57 @@ def init(self) -> None:
                              exc_info=True)
         self._init_argus_resource()
 
+    @cached_property
+    def argus_client(self) -> ArgusSCTClient | None:
```

Comment:
you might consider moving this to `TestConfig` class, since we are creating this client multiple times
across different files/classes, testet.py / nemesis.py / common.py

Reply:
Would it be easily accessible across modules, especially in `sct.py`? Maybe Fabio's idea with a central module would work better here?

Reply:
TestConfig is central, and unique across (Singleton) the whole SCT so that should fit the pattern you are looking for. creating a new Singleton also doable, would be the same from POV.

Reply:
I think with current pattern we may be able to get away with just static methods with a cached class attribute containing the credentials (or client), so that may work even better